tests/unit/style/atomic_refcell.rs

```diff
@@ -12,9 +13,67 @@ struct Bar {
     f: Foo,
 }
 
+impl Default for Bar {
+    fn default() -> Self {
+        Bar { f: Foo { u: 42 } }
+    }
+}
+
```

Comment:
We should probably test this in parallel conditions? I guess the style code is enough, but still...